recipes-core/install-on-emmc/files/install-on-emmc.sh

```diff
@@ -137,13 +138,16 @@ if ! test -b ${EMMC_DEV}p1; then
 		sleep 1
 	done
 fi
+
+BOOT_DEV_UUID=$(sfdisk --part-uuid ${BOOT_DEV} ${ROOT_PART})
+
 mount ${EMMC_DEV}p1 /mnt
 mount -o bind /dev /mnt/dev
 mount -t proc proc /mnt/proc
-chroot /mnt sh -c ' \
-    /usr/share/regen-rootfs-uuid/regen-rootfs-uuid.sh
+chroot /mnt sh -c " \
```

Comment:
if I am not mistaken there are no error checks throughout this code
I worry that we could end-up with a corrupted install

Reply:
Yup, it could - and it did already in some cases. But that is a pre-existing issue.